<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battleship</class>
 <widget class="QMainWindow" name="Battleship">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Battleship</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>219</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="myGridLayout"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="enemyGridLayout"/>
   </widget>
   <widget class="QPushButton" name="serverButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clientButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="logTextWindow">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
